src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/Thread.java

```diff
@@ -76,23 +72,6 @@ public int suspendFlags() {
     return (int) suspendFlagsField.getValue(addr);
   }
 
-  public boolean isExternalSuspend() {
-    return (suspendFlags() & EXTERNAL_SUSPEND) != 0;
-  }
-
-  public boolean isExtSuspended() {
```

Comment:
A new comment to avoid the "resolved" conversation. I'm still not sure you can just delete this API from the SA. @sspitsyn do you know what the obligations are with respect to the S.A here?

Reply:
The SA do not have the HandshakeState class implemented, so there is a lot a work to make this flag work.